src/formatters/stylishFormatter.ts

```diff
@@ -26,15 +26,24 @@ export class Formatter extends AbstractFormatter {
             return "\n";
         }
 
-        const fileName        = failures[0].getFileName();
+        const outputLines: Array<string> = [];
+
         const positionMaxSize = this.getPositionMaxSize(failures);
         const ruleMaxSize     = this.getRuleMaxSize(failures);
 
-        const outputLines = [
-            fileName,
-        ];
+        let currentFile: string;
 
         for (const failure of failures) {
+            // File name for each new file
+            const fileName = failure.getFileName();
+            if (currentFile !== fileName) {
+                if (currentFile) {
+                    outputLines.push("");
```

Comment:
It wouldn't be. It would be only for the first file, which is the case I'm trying to catch here, to avoid adding an empty line.
It's either that, or adding the line even at the first time, and then unshifting it out of the outputLines before returning it. Although then there is a need to checked whether the array is not empty and etc.